OnlineShop/OnlineShop/Models/ScaffDir/ShopDbContext.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.EntityFrameworkCore;
+using online_s.Configurations;
+
+namespace online_s.ScaffDir;
+
+public partial class ShopDbContext : DbContext
+{
+    public ShopDbContext()
+    {
+    }
+
+    public ShopDbContext(DbContextOptions<ShopDbContext> options)
+        : base(options)
+    {
+    }
+
+    public virtual DbSet<Adress> Adresses { get; set; }
+
+    public virtual DbSet<Brand> Brands { get; set; }
+
+    public virtual DbSet<Cart> Carts { get; set; }
+
+    public virtual DbSet<Category> Categories { get; set; }
+
+    public virtual DbSet<Color> Colors { get; set; }
+
+    public virtual DbSet<OrderProduct> OrderProducts { get; set; }
+
+    public virtual DbSet<ProductVariant> ProductVariants { get; set; }
+
+    public virtual DbSet<Order> Orders { get; set; }
+
+    public virtual DbSet<Product> Products { get; set; }
+
+    public virtual DbSet<Review> Reviews { get; set; }
+
+    public virtual DbSet<Section> Sections { get; set; }
+
+    public virtual DbSet<Size> Sizes { get; set; }
+
+    public virtual DbSet<User> Users { get; set; }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.ApplyConfiguration(new AdressConfiguration());
+
+        modelBuilder.ApplyConfiguration(new BrandConfiguration());
```

Comment:
You an replace this `modelBuilder.ApplyConfiguration(new ColorConfiguration());` boilerplate using ApplyConfigyrationFromAssembly